src/test/java/picard/arrays/illumina/VcfToAdpcTest.java

```diff
@@ -0,0 +1,30 @@
+package picard.arrays.illumina;
+
+import htsjdk.samtools.util.IOUtil;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.arrays.VcfToAdpc;
+
+import java.io.File;
+import java.io.IOException;
+
+public class VcfToAdpcTest {
+
+    private static final File TEST_DATA_DIR = new File("testdata/picard/arrays/illumina");
+    private static final File TEST_VCF = new File(TEST_DATA_DIR, "TestVcfToAdpc.vcf");
+    private static final File TEST_EXPECTED_ADPC_BIN_FILE = new File(TEST_DATA_DIR, "TestIlluminaAdpcFileWriter.adpc.bin");
```

Comment:
how did you generate the expected file?

Reply:
I generated it with the tool itself.
- I reached out to the lab to see if they could generate one (to verify the header contents) but they have no idea about it (and reached out to Illumina who seemed to think it was an older file, used for Illumina Golden Gate assays - I think they're wong)
- Further I needed a small file for testing.

Reply:
OK. well...I guess that's all we can hope for then.